client/src/interface/profile/Profile.ts

```diff
@@ -0,0 +1,15 @@
+type image = {
+  url: string;
+  public_id: string;
```

Comment:
```suggestion
  publicId: string;
```

Please resort to camelCase when using JS